### PR TITLE
Create MES5324.yml

### PR DESCRIPTION
**Pull request by GitRost**
Fix Mgmt interfice

### CODE REVIEW

File: device-types/Eltex/MES5324.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+manufacturer: Eltex
+model: MES5324
+slug: eltex-mes5324
+part_number: MES5324
+is_full_depth: false
```
Comment (danner26) [inline device-types/Eltex/MES5324.yml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Eltex/MES5324.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+console-ports:
+  - name: con0
+    type: rj-45
+power-ports:
+  - name: PS0
+    type: iec-60320-c14
+    maximum_draw: 107
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 107
```
Comment (danner26) [inline device-types/Eltex/MES5324.yml:79]: It appears that the PSUs on this device are modular. If that is the case, then you should be modelling these under `module-bays` and the applicable modular PSUs should be added under the module-types folder

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Did you just ignore both of my comments and request a re-review?

**Comment by GitRost** [comment]
> Did you just ignore both of my comments and request a re-review?

Sorry if I inserted something in the wrong place and clicked the wrong thing.
I don’t know how to use github, and nothing is intuitive here.

**Pull request closed without merging**